pkg/manifest/manifest.go

```diff
@@ -384,6 +390,18 @@ func (manifest *manifest) WritePlatformConfig(out io.Writer) error {
 	return nil
 }
 
+func (manifest *manifest) WriteResourceLimitsConfig(out io.Writer) error {
+	bytes, err := yaml.Marshal(manifest.ResourceLimits)
+	if err != nil {
+		return util.Errorf("Could not write resource limits for %s: %s", manifest.ID(), err)
```

Comment:
Should you distinguish this error message from the one on line 400 so we can differentiate between marshaling and writing errors more easily? Or do you think including the `err` at the end will be sufficient for that?